### ISSUE TITLE
Хранить на корпоративном бакете S3 

### ISSUE THREAD

**Issue by TimofeevVM** [polina4096/voices#17]
Дать возможность сохранять аудиофайлы на корпоративном s3. Это дешевый сервис, в котором можно "вечно" хранить файлы и не перезагружать репозиторий бинарными файлами

**Comment by polina4096** [comment]
рассмотрю эту возможность

**Comment by inyutin-maxim** [comment]
@TimofeevVM Хотел уточнить вы имеете ввиду Amazon или что то приватное?

**Comment by TimofeevVM** [comment]
> @TimofeevVM Хотел уточнить вы имеете ввиду Amazon или что то приватное?

Да, aws s3, но в целом Amazon задал стандарт, поэтому s3-хранилище предоставляет многие провайдеры (Yandex Object Storage, MTS, VK Cloud) и так же есть self-hosted решения (MinIO, Ceph)

**Comment by inyutin-maxim** [comment]
Меня как раз интересуют self-hosted решения
